tests/unittests/caffe2ImporterTest.cpp

```diff
@@ -1013,3 +1013,58 @@ TEST(caffe2, batchBoxCox) {
     EXPECT_FLOAT_EQ(result.raw(i), OH.raw(i));
   }
 }
+
+// Test loading a EQ operator with 1D inputs.
+TEST(caffe2, EQ1D) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetDescFilename("tests/models/caffe2Models/eq_op_net.pbtxt");
+  std::string NetWeightFilename(
+      "tests/models/caffe2Models/empty_init_net.pbtxt");
+
+  Placeholder *output;
+  Context ctx;
+
+  // Input tensors.
+  const size_t kDataSize = 10;
+  Tensor X(ElemKind::FloatTy, {kDataSize});
+  Tensor Y(ElemKind::FloatTy, {kDataSize});
+  auto XH = X.getHandle();
+  auto YH = Y.getHandle();
+
+  // Fill inputs with same random values.
+  XH.randomize(-3.0, 3.0, mod.getPRNG());
+  for (size_t i = 0; i < kDataSize; ++i) {
+    YH.at({i}) = XH.at({i});
+  }
+
```

Comment:
Since no real run is needed, don't need to assign value to the inputs.

Reply:
@beicy can you outline the guideline on what needs and what does not need to be tested in this/and onnxLoader tests?
So it would be clear on what really should be in this test.